app/models/glossary_term.rb

```diff
@@ -1,10 +1,18 @@
 class GlossaryTerm < AbstractModel
   require "acts_as_versioned"
 
-  belongs_to :thumb_image, class_name: "Image", foreign_key: "thumb_image_id"
+  belongs_to(:thumb_image,
+             class_name: "Image",
+             foreign_key: "thumb_image_id",
+             inverse_of: :best_glossary_terms)
   belongs_to :user
   belongs_to :rss_log
+  # Rubocop would rather this was a has_many :through, but that
+  # requires a migration process as detailed here:
+  # chrisrolle.com/en/blog/migration-path-from-habtm-to-has_many-through
+  # rubocop:disable Rails/HasAndBelongsToMany
```

Comment:
Should this be disabled in .rubocop.yml?

Reply:
Yes.